pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -603,7 +606,7 @@ func addEndpointSubset(subsets []v1.EndpointSubset, pod *v1.Pod, epa v1.Endpoint
 		})
 		readyEps++
 	} else { // if it is not a ready address it has to be not ready
-		klog.V(5).Infof("Pod is out of service: %s/%s", pod.Namespace, pod.Name)
+		klog.FromContext(ctx).V(5).Info("Pod is out of service", "pod", klog.KRef(pod.Namespace, pod.Name))
```

Comment:
Please also change this elsewhere.

```suggestion
		klog.FromContext(ctx).V(5).Info("Pod is out of service", "pod", klog.KObj(pod))
```